Packs/SalesforceIndicators/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Salesforce Indicators
+- Removed duplicated params.
```

Comment:
```suggestion
- Fixed an issue where the *Indicator Reputation* parameter was duplicated in the integration configuration.
```

Reply:
@tomneeman151293 just commit these changes and you're good to go

Reply:
And the `Fetch indicators` also

Reply:
Fixed